pkg/kubelet/dockertools/docker.go

```diff
@@ -316,7 +316,11 @@ func (d *dockerContainerCommandRunner) PortForward(podInfraContainerID string, p
 	}
 
 	containerPid := container.State.Pid
-	// TODO use exec.LookPath for socat / what if the host doesn't have it???
+	// TODO what if the host doesn't have it???
```

Comment:
agree
let's keep this for another PR for this specific point
I was only touching this part to remove frustration of not knowing the cause of the error.